concordium-sdk/src/main/java/com/concordium/sdk/transactions/BlockItem.java

```diff
@@ -16,6 +21,18 @@ private BlockItem(AccountTransaction accountTransaction) {
         this.accountTransaction = accountTransaction;
     }
 
+    @Nullable
```

Comment:
I suggest moving this just below the comment to avoid a 'dangling comment warning'

Reply:
Ok.